task/02-numbers-tasks.js

```diff
@@ -202,7 +201,13 @@ function roundToPowerOfTen(num, pow) {
  *   17 => true
  */
 function isPrime(n) {
-    throw new Error('Not implemented');
+    for (let i = 2; i <= Math.sqrt(n);) {
+		if (n % i == 0) {
```

Comment:
it is necessary to observe indents (not too much)
https://javascript.info/coding-style